src/types/ProposalFieldValue.ts

```diff
@@ -29,6 +30,9 @@ export const proposalFieldValueWriteSchema: JSONSchemaType<ProposalFieldValueWri
 			value: {
 				type: 'string',
 			},
+			isValid: {
```

Comment:
I'm marking this on this line but it's unrelated -- we need to update the openApi spec to add `isValid` there as well.